src/dcqc/parsers.py

```diff
@@ -45,19 +45,16 @@ def create_files(self) -> Iterator[tuple[int, File]]:
                 file.stage(destination, overwrite=True)
             yield index, file
 
-    def create_targets(self, stage_files: bool = True) -> Iterator[Target]:
+    def create_targets(self) -> Iterator[Target]:
         for index, file in self.create_files():
-            if stage_files:
-                file.stage()
             yield Target(file, id=f"{index:04}")
```

Comment:
I didn't notice this last time, but why 04 and not 05 or 03?

Reply:
I feel that the chance that there is more than 999 files in a workflow run is non-trivial, but I think it's rare that a workflow run would have more than 9,999 files. And because I'm iterating through the CSV file, I don't know how many rows there are ahead of time.